crates/decoder/src/dbin.rs

```diff
@@ -123,12 +123,12 @@ pub struct DbinHeader {
     /// File format version, the next single byte after the 4 [`DbinMagicBytes`]
     version: Version,
     /// Content type like 'ETH', 'type.googleapis.com/sf.ethereum.type.v2.Block'
-    content_type: String,
+    pub content_type: String,
```

Comment:
This is always a bit of a "code smell" when a single field on a container is being made public. I suggest adding a method within `impl DbinHeader {}` to provide an api for accessing the field.